bot/cogs/error_handler.py

```diff
@@ -89,6 +95,28 @@ def __init__(self, bot: Bot):
         else:
             return self.bot.get_command("help")
 
+    async def try_silence(self, ctx: Context) -> bool:
+        """
+        Attempt to invoke the silence or unsilence command if invoke with matches a pattern.
+
+        Respecting the checks if:
+                                  invoked with `shh+` silence channel for amount of h's*2 with max of 15.
+                                  invoked with `unshh+` unsilence channel
```

Comment:
This is indented way too far.

Reply:
Was aiming for a list like look here (under the colon).
Should it just be a 2-4 spaces?

Reply:
Yes, 4 spaces should be fine. If you think it helps you can prepend an asterisk or number them.

Reply:
Went with an asterisk and a space like in other docstrings in the module.